src/screens/AddParticipant/index.js

```diff
@@ -55,8 +55,8 @@ const AddParticipant = ({navigation, route}) => {
   };
   const filterDataUser = (dataUsers) => {
     return dataUsers.filter((itm1) => {
-      return !Object.keys(participants).some((itm2) => {
-        return itm1.user_id === itm2;
+      return !participants.some((itm2) => {
+        return itm1.user_id === itm2.user_id;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The logic of filtering participants has been changed from using `Object.keys(participants)` to directly using `participants`. This change assumes that `participants` is an array. If `participants` is not guaranteed to be an array, this could cause a runtime error.
```diff
- return !Object.keys(participants).some((itm2) => {
+ return !participants.some((itm2) => {
    return itm1.user_id === itm2.user_id;
});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->